### PR TITLE
Some quick notes

### PR DESCRIPTION
**Pull request by ghost**
As this is going to be for training, I'll try to document everything in the codebase, this is a start.

### CODE REVIEW

**Comment by ghost** [comment]
@arkgil ! Love this implementation. So simple. I knew that implementing a custom Cowboy handler was overkill at this stage.

File: README.md [hunk 1]
```diff
@@ -70,3 +74,51 @@ If you look at the logs again:
 By default file is read and sent in chunks of 5000 bytes. You can change this number by modifying
 `:upload.:download_chunk_size` configuration variable.
 
+
+----
+
+#### Bryan notes
```
Comment (arkgil) [inline README.md]: Could we move that to `NOTES.md` or something like that?

File: README.md [hunk 2]
```diff
@@ -16,10 +16,14 @@ by setting `:upload.:port` configuration value.
...
 
+Generate some sample data : 
+
+`seq 1 100000 | while read i ; do echo "line $1 : foo bar baz basho" >> /tmp/upload.garbage.txt ; done`
+
 You can upload files via curl:
 
 ```bash
-$ curl -X PUT http://localhost:8080/uploads/your-file-name --data-binary @path/to/your/file
+curl -X PUT http://localhost:8080/uploads/garbage.txt --data-binary @/tmp/upload.garbage.txt
```
Comment (arkgil) [inline README.md:28]: Could you change the path for GET request below as well?

Comment (ghost) [reply README.md:28]: done

**Comment by ghost** [comment]
@arkgil - anything else before merge ?

**Review by arkgil — APPROVED** [review]

**Comment by arkgil** [comment]
@bryanhuntesl could you squash/give descriptive names to last two commits?

**Comment by ghost** [comment]
@arkgil - you choose to squash them when you merge. I can't do it without having to create new branch.

**Comment by ghost** [comment]
And. Like I mentioned. 4 day (14hr days) hackathon.

**Pull request merged**